cdisc_rules_engine/data_service/sql_interface.py

```diff
@@ -113,13 +117,17 @@ def fetch_all(self) -> List[Optional[Any]]:
         self._last_results.clear()
         return results
 
-    def create_table(self, schema: SqlTableSchema) -> None:
+    def create_table(self, schema: SqlTableSchema) -> SqlTableSchema:
```

Comment:
Why would the name change? Surely you only want to edit the `hash` then nothing else has to ever worry about whether the name is prefixed or not, as long as they use the hash. It's why I added table hashes, knowing the prefixes were planned in the distance